--- ./src/test/java/org/apache/commons/math3/genetics/UniformCrossoverTest.java.bak	2022-06-17 15:09:34.000000000 -0700
+++ ./src/test/java/org/apache/commons/math3/genetics/UniformCrossoverTest.java	2022-06-20 17:02:37.000000000 -0700
@@ -24,6 +24,7 @@
 import org.apache.commons.math3.exception.DimensionMismatchException;
 import org.apache.commons.math3.exception.MathIllegalArgumentException;
 import org.apache.commons.math3.exception.OutOfRangeException;
+import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
@@ -32,8 +33,10 @@
     private static final List<Integer> p1 = new ArrayList<Integer>(LEN);
     private static final List<Integer> p2 = new ArrayList<Integer>(LEN);
 
-    @BeforeClass
-    public static void setUpBeforeClass() {
+    @Before
+    public void setUpBeforeClass() {
+        p1.clear();
+        p2.clear();
         for (int i = 0; i < LEN; i++) {
             p1.add(0);
             p2.add(1);
